.travis.yml

```diff
@@ -28,6 +33,8 @@ before_script:
 
   - export NODE_PATH="$(pwd)/node_modules"
 
+  - if [[ "$IOJS_VERSION" != "" ]]; then wget https://iojs.org/dist/v${IOJS_VERSION}/iojs-v${IOJS_VERSION}-linux-x64.tar.xz && tar xvf iojs-v${IOJS_VERSION}-linux-x64.tar.xz && pwd; fi
```

Comment:
To get absolute folder path that I can put into `NODE_BIN` env variable. Maybe however I need to:

1. store `pwd` result into some variable
2. build NODE_BIN value as `pwd result` + `/iojs-v${IOJS_VERSION}-linux-x64/bin/node`